src/Hamelin/FileSystem/Physical/PhysicalDirectory.cs

```diff
@@ -12,17 +13,19 @@ internal class PhysicalDirectory(string path) : IDirectory
     public IFile GetFile(string name) => new PhysicalFile(Path.Combine(AbsolutePath, name));
     public IDirectory GetDirectory(string name) => new PhysicalDirectory(Path.Combine(AbsolutePath, name));
 
-    public IEnumerable<IFile> GetFiles()
+    public IEnumerable<IFile> GetFiles(string searchPattern = "*.*")
```

Comment:
```suggestion
    public IEnumerable<IFile> GetFiles(string searchPattern = "*")
```
`*.*` will match all files with a name and extension part, but sometimes files only have a name or the name begins with a `.` - `*` should match all files.

Reply:
Oh yeah, good point. Dotfiles are hidden by default on Mac, and files without extensions make me feel uncomfortable 😁